bftengine/src/bcstatetransfer/BCStateTran.cpp

```diff
@@ -1110,12 +1110,19 @@ void BCStateTran::setEraseMetadataFlagImpl() { psd_->setEraseDataStoreFlag(); }
 
 void BCStateTran::setReconfigurationEngineImpl(std::shared_ptr<ClientReconfigurationEngine> cre) { cre_ = cre; }
 
+template <typename MSG>
+void BCStateTran::stateTransferMsgDeleter(const MSG *m) {
```

Comment:
A better name would be freeStateTransferMsg.
The name you gave this function is a noun rather than imperative.

Reply:
done, thanks